lnrpc/verrpc/verrpc.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package verrpc;
+
+option go_package = "github.com/lightningnetwork/lnd/lnrpc/verrpc";
+
+service Versioner {
+    rpc GetVersion (VersionRequest) returns (Version);
+};
+
+message VersionRequest {
+};
+
+message Version {
+    /// A verbose description of the daemon's commit.
+    string commit = 1;
+
+    /// The SHA1 commit hash that the daemon is compiled with.
+    string commit_hash = 2;
+
+    /// The semantic version.
+    string version = 3;
+
+    /// The major application version.
+    uint32 app_major = 4;
+
+    /// The minor application version.
+    uint32 app_minor = 5;
+
+    /// The application patch number.
+    uint32 app_patch = 6;
+
+    /// The application pre-release modifier, possibly empty.
+    string app_pre_release = 7;
+
+    /// The list of build tags that were supplied during comilation.
+    repeated string build_tags = 8;
```

Comment:
Looking at how this field is used in https://github.com/lightninglabs/loop/pull/181 with the string matching and all, shouldn't we replace this with a compatibility flagged enum?